packages/medusa/src/loaders/module.ts

```diff
@@ -20,22 +20,36 @@ export default async ({
 
   for (const resolution of Object.values(moduleResolutions)) {
     try {
+      if (!resolution.resolutionPath) {
+        container.register({
+          [resolution.definition.registrationName]: asFunction(
+            () => false
+          ).singleton(),
+        })
+
+        continue
+      }
+
       const loadedModule = await import(resolution.resolutionPath!)
```

Comment:
thought: can this fail? e.g. if the user has not specified anything but hasn't installed the `defaultPackage` in this scenario import would not be able to get the package. It would be helpful to show an error message with:

```
We could not resolve [resolutionPath] for module [moduleKey]. You may need to run yarn add [resolutionPath].
```